test-run.py

```diff
@@ -88,16 +86,16 @@ def main_loop_parallel():
                      schema='tr_text')
     randomize = True
 
-    task_groups = lib.worker.get_task_groups()
+    task_groups = get_task_groups()
```

Comment:
Maybe there is sense in visual differentiation of a 'thread / worker' code from the 'main process / dispatcher' code. I modestly vote for calling lib / worker code from outside of lib as 'lib.worker.<...>' or at least 'worker.<...>'.